### DISCUSSION TITLE
unable to compile gio

### DISCUSSION THREAD

**Discussion by sheshnath-at-knoldus** [gtk-rs/gtk-rs-core#1442] [Q&A]
I am unable to build gio in raspberry pi b+ model

**Comment by sdroege** [comment]
How does it fail?

**Reply by sheshnath-at-knoldus** [reply]
it does not fail it stuck at building gio

**Reply by sdroege** [reply]
That sounds like a resource problem. Maybe you're running out of RAM and everything gets stuck

**Reply by sheshnath-at-knoldus** [reply]
yes it may be a reason
